webpack/services/api/index.js

```diff
@@ -99,6 +99,15 @@ class ForemanTasksApi extends Api {
 
 export const foremanTasksApi = new ForemanTasksApi();
 
+class ForemanUIAPI extends Api {
```

Comment:
This name seems a like a contradiction to me.  Maybe we can name it "ForemanEndpoint", "ForemanUIEndpoint" or something?  To me it's confusing to call something both a UI and an API.